DB Schema/user_document.json5

```diff
@@ -0,0 +1,7 @@
+user document
+{
```

Comment:
We should add title and surname also, right?

Reply:
Done, After Discussion with atul, I have removed name and added Firstname and Lastname instead, but title field is not necessary as of now